index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./style.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto" />
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins" />
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Crete Round" />
+        
+    </head>
+    <body>
+        <section class="section-1">
+            <nav class="nav-bar">
+                    <div class="logo-name">
+                            <p>DANIEL OCHUBA</p>
+                    </div>
+                    <div class="menu-button">
+                        <div class="toggle"></div>
+                        <div class="toggle"></div>
+                        <div class="toggle"></div>
+                    </div>
+                    
+                </nav>
+        </section>
```

Comment:
- [ ] Kindly make sure this container is wrapped into a `header` tag instead of a `section` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info